assets/css/startertheme.css

```diff
@@ -0,0 +1,123 @@
+.project-grid {
+  margin-left: 100px;
```

Comment:
This class is unneeded as you are using bootstrap. It's better to use `rem` instead of `px`. You can use the bootstrap classes `ml-# mr-# mt-#` where # is a number from 1-5 corresponding to a rem unit. And ml stands for margin-left, mr stands for margin-right, etc. Check out the bootstrap docs for more info.